### PR TITLE
Adds GraphQL-Explorer to WP-GraphiQL in the way gatsby is using it.

### PR DESCRIPTION
**Pull request by henrikwirth**
Solving this feature request: #14 

So here is a working example already, but with some issues. Queries are generated via URL decoding and this needs to be fixed. Also the styling is off and needs to be adapted.

Tasks:

- [x] Add explorer
- [x] Add styling for explorer
- [x] Make sure url encoding/decoding works properly

### CODE REVIEW

**Comment by henrikwirth** [comment]
@jasonbahl This works now. The only litte problem is that the URI decoding ends up have no line-breaks. So when you make a query and then use the same generated URI you will see it is without line-breaks. But it works and Prettify makes in pretty again. Maybe you have an idea about the URI-Decoding.

**Comment by jasonbahl** [comment]
@henrikwirth I'm checking this out now. Thanks for the PR!

File: assets/app/src/App.js [hunk 1]
```diff
@@ -1,42 +1,335 @@
...
+		: true
+		: window.localStorage
+		? window.localStorage.getItem(`graphiql:graphiqlExplorerOpen`) !== `false`
+		: true
+
+
+class App extends React.Component {
+	state = {
+		schema: null,
+		query: DEFAULT_QUERY,
```
Comment (jasonbahl) [inline assets/app/src/App.js:161]:
```suggestion
		query: print(parse(DEFAULT_QUERY)),
```

File: assets/app/src/App.js [hunk 2]
```diff
@@ -1,42 +1,335 @@
 import React from 'react';
-import GraphiQL from 'graphiql';
+
+import GraphiQL from "graphiql"
+import GraphiQLExplorer from "graphiql-explorer"
+import {getIntrospectionQuery, buildClientSchema, parse} from "graphql"
```
Comment (jasonbahl) [inline assets/app/src/App.js]:
```suggestion
import {getIntrospectionQuery, buildClientSchema, parse, print} from "graphql"
```

**Review by jasonbahl — CHANGES_REQUESTED** [review]
Yo! I got it working. 2 small changes, then run `yarn build` again and we can get this merged!

**Review by jasonbahl — APPROVED** [review]
Thanks! This is a fantastic addition to the plugin!

**Pull request merged**